content/docs/start/experiments.md

```diff
@@ -194,7 +193,7 @@ Storage, HTTP, HDFS, etc.). The Git remote is often a central Git server
 
 </details>
 
-Experiments that have not been made persistent will not be stored or shared
+Experiments which have not been made persistent will not be stored or shared
```

Comment:
Restrictive clause (I know, this is an original change. I just never got to this one file before).

To recap, could be a personal style choice, but reverting since it's already published this way and for consistency with previous reverts in this PR.

```suggestion
Experiments that have not been made persistent will not be stored or shared
```